braindecode/preprocessing/preprocess_classes.py

```diff
@@ -0,0 +1,155 @@
+"""Preprocessor objects based on mne methods.
+"""
+
+# Author: Bruna Lopes <brunajaflopes@gmail.com>
+#
+# License: BSD-3
+
+import mne.io
+from braindecode.util import _update_moabb_docstring
+from braindecode.preprocessing import Preprocessor
+
+
+class Resample(Preprocessor):
+    doc = """
+    Subclass of Preprocessor to perform resampling using mne.Epochs/mne.io.Raw's resample method.
+
+    Parameters
+    ----------
+    kwargs:
+        Keyword arguments to be passed to mne.Epochs's resample method.
+    """
+    try:
+        base_class = mne.io.Raw.resample
+        _doc__ = _update_moabb_docstring(base_class, doc)
+    except ModuleNotFoundError:
+        pass
+
+    def __init__(self, **kwargs):
+        # Ignore "fn" parameter -> the only preprocessing that is going to
+        # be used here is mne.Epochs's resample
+        # Removing this string parameter, because it's going to execute
+        # directly the said preprocess
+
+        fn = 'resample'
+        self.fn = fn
+        self.kwargs = kwargs
+
+        super().__init__(fn, **kwargs)
+
+
+class DropChannels(Preprocessor):
+    doc = """
+    Subclass of Preprocessor to drop specific channels using mne.Epochs/mne.io.Raw's method.
+
+    Parameters
+    ----------
+    kwargs:
+        Keyword arguments to be passed to mne.Epochs's resample method.
```

Comment:
```suggestion
        Keyword arguments to be passed to mne.Epochs's drop_channels method.
```
You could add the link to MNE doc also.